### PR TITLE
Update issue templates to include bug report

### PR DESCRIPTION
**Pull request by cmgorton**
Will be adding an additional config file to add links to forem.dev for discussion and feature requests.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (jdoss) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: I would tailor these to the dependencies of the selfhost repo. We don't really care what their browser version is.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (jdoss) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: This can be totally removed. We don't care about smartphone info.

**Review by jdoss — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (cmgorton) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: Sounds good. I will remove it.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (cmgorton) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: Ok would you say adding the OS (Linux or MacOS) would be helpful and then asking which cloud provider they are using? 

So something like:
**OS:**
- Linux
- MacOS

**Cloud Provider**
- AWS
- Google Cloud
- DigitalOcean
- Other

Comment (jdoss) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: Yep these would be super great. LGTM!

Comment (cmgorton) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ok I added them!

**Review by jdoss — APPROVED** [review]
LGTM!

**Pull request merged**